cpp/example_code/transcribe/README.md

```diff
@@ -0,0 +1,43 @@
+# Amazon Transcribe C++ SDK code examples
```

Comment:
Update this README to match the current [service-level template](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template). Get it as close as possible--I know this is sort of a special-case example and you likely want to keep the specific build/setup instructions, but I think getting the README as similar to the standard as you can will be the best customer experience.

Reply:
ReadMe has been updated.